src/js/account/components/AccountProfile.tsx

```diff
@@ -56,9 +48,13 @@ export default function AccountProfile() {
                     <h3>
                         {handle}
                         {administrator_role && (
-                            <AdministratorTag key="administrator" color="purple">
+                            <Label
+                                className={`capitalize text-[${theme.fontSize.md}] ml-auto`}
```

Comment:
Tailwind is the theme.
```suggestion
                                className="capitalize text-md ml-auto"
```